core/libdeno/exceptions.cc

```diff
@@ -200,6 +200,7 @@ void HandleException(v8::Local<v8::Context> context,
   std::string json_str = EncodeExceptionAsJSON(context, exception);
   CHECK_NOT_NULL(d);
   d->last_exception_ = json_str;
+  d->last_exception_handle_.Reset(isolate, exception);
```

Comment:
I wonder if there will be problems with the ASAN test. We might have to reset the handle in the destructor if so.

Reply:
Fixed -- added Reset() to the destructor.